bokeh/server/tornado.py

```diff
@@ -188,14 +190,30 @@ def __init__(self, applications, prefix, hosts,
 
             all_patterns.extend(app_patterns)
 
+            # add a per-app static path if requested by any app handler
+            # (there should be at most one that requests a static path)
```

Comment:
We have a views/static_handler.py with a custom subtype of StaticFileHandler, and that class claims it exists "because we might want to serve static files from multiple paths in the future" - however it doesn't appear to actually implement that yet ;-)